ui/src/onboarding/components/OnboardingButtons.tsx

```diff
@@ -51,31 +52,24 @@ class OnboardingButtons extends PureComponent<Props> {
   }
 
   public render() {
-    const {nextButtonText, nextButtonStatus} = this.props
+    const {nextButtonText, nextButtonStatus, className} = this.props
     return (
-      <div className={this.className}>
-        <div className="wizard--button-bar">
-          {this.backButton}
-          <Button
-            color={ComponentColor.Primary}
-            text={nextButtonText}
-            size={ComponentSize.Medium}
-            type={ButtonType.Submit}
-            testID="next"
-            ref={this.submitRef}
-            status={nextButtonStatus}
-            tabIndex={0}
-          />
-        </div>
+      <Overlay.Footer className={className}>
```

Comment:
the previous implementation accounted for whether `className` was undefined. Should we still maintain that check here by having a default empty string or would `className` never be undefined?

Reply:
If it is passed in as `undefined` than nothing happens